phalcon/forms/form.zep

```diff
@@ -712,7 +684,11 @@ class Form extends Injectable implements \Countable, \Iterator
 	public function rewind() -> void
 	{
 		let this->_position = 0;
-		let this->_elementsIndexed = array_values(this->_elements);
```

Comment:
If you create a form without any element, and iterate that, php warning will be shown, because this->_elements is null. I added some code to prevent that. https://github.com/phalcon/cphalcon/blob/3.0.x/tests/unit/Forms/FormTest.php#L77